src/MoBi.Presentation/Nodes/ModuleConfigurationNode.cs

```diff
@@ -9,7 +10,7 @@ public class ModuleConfigurationNode : AbstractNode<ModuleConfigurationDTO>
       public ModuleConfigurationNode(ModuleConfigurationDTO moduleConfiguration)
          : base(moduleConfiguration)
       {
-         Id = moduleConfiguration.Module.Id;
+         Id = Guid.NewGuid().ToString();
```

Comment:
explain why are you not using the Id of the module here. Isn't this supposed to be unique? Very suspicious

Reply:
The module Id is unique, but the module can be used any number of times within the simulation explorer. I'll add a comment.

Reply:
Well, I'm not sure what led me to change that originally, but after thinking about it more and testing, this isn't quite right. The modules are clones of the project module, so they should have unique Id's in the simulation explorer.

Reply:
Exactly. This is not accurate otherwise